client/my-sites/post-type-list/index.jsx

```diff
@@ -153,6 +154,7 @@ class PostTypeList extends Component {
 						siteId={ siteId }
 						query={ { ...query, page } } />
 				) ) }
+				{ siteId && <QueryPostTypes siteId={ siteId } /> }
```

Comment:
Regarding labels I agree simple verbs would be better.

Regarding the queries, it seems multiple queries should have a way to not run if there's another that was already run by a parent within a certain interval. But it still seems that QueryPostTypes is something I would run in my `main.jsx` component, not on the list (which would run the QueryPosts).